references.bib

```diff
@@ -519,15 +544,15 @@ @misc{abel2012agda
   @misc{danielsson2011agda,
```

Comment:
That is not the correct bib entry for the standard library!

Reply:
I had another reference for the standard library. I fixed them now and kept the version as v1.7.